src/main/index.js

```diff
@@ -122,17 +122,22 @@ function init () {
 }
 
 function delayedInit () {
+  if (app.isQuitting) return
```

Comment:
Fixes exception sometimes seen in terminal during development, where `delayedInit` gets called while app is quitting